util/configuration.py

```diff
@@ -281,6 +297,9 @@ def update(cls):
                         line = 'rcon.password={}\n'.format(cls.password)
                     
                     fileOut.write(line)
+        else:
+            from .syslog import log
```

Comment:
Has this portion of functionality been tested fully? Because if memory serves, when importing syslog into configuration it causes an error.

Reply:
Considering that this was from the RCON stuff, I would think it was fully tested... However, if memory serves, there really shouldn't be any way for the user to get to that path anyway. Besides, if there is no server.properties file, they won't be able to start the server, either.